### DISCUSSION TITLE
When will we be releasing version 1.0.0?

### DISCUSSION THREAD

**Discussion by Camerooooon** [JakeRoggenbuck/auto-clock-speed#306] [General]
Hi everyone, I just wanted to briefly discuss our plans for versioning for the future.

So we will continue incrementing our 0.1.*v version system until we release our 1.0.0v version. We will be releasing 1.0.0v when we believe our APIs are stable enough to be released to public desktop package managers (apt, pacman, etc). 1.0.0v will be a big milestone for us and the auto clock speed team is incredibly excited to keep looking to the future for ACS.

**Comment by Camerooooon** [comment]
Here is our roadmap for full release!

- [ ] 0.1.10 All features are added to auto clock speed. This release might take a bit longer as we add the last of our ideas to acs.
- [ ] 0.1.11 Refactor major portions of code for release. Stabilize and fix bugs for all systems.
- [ ] 1.0.0 Acs is determined to be stable enough to be release to public repositories. We will now focus on expanding our community.

**Comment by JakeRoggenbuck** [comment]
We will also be creating a discord server when we hit 1.0
